site/gatsby-site/src/utils/Translator.js

```diff
@@ -125,7 +127,24 @@ class Translator {
   async run() {
     await this.mongoClient.connect();
 
-    const reports = await this.mongoClient.db('aiidprod').collection(`reports`).find({}).toArray();
+    let reportsQuery = {};
+
+    if (this.submissionDateStart) {
+      this.reporter.log(
+        `Translating incident reports submitted after [${this.submissionDateStart}]`
+      );
+      reportsQuery = { date_submitted: { $gte: new Date(this.submissionDateStart) } };
```

Comment:
We should probably crash/throw + return error if we don't parse the date correctly (not sure how `$gte` will like null :-) )

Reply:
The `if (this.submissionDateStart)` statement avoids `null` values but I added a check for another type of invalid string value like `blah blah` just in case. Thanks for your input!